src/launchHandler.py

```diff
@@ -101,11 +63,17 @@ def launchCallback(data):
     if fileName not in launchedFiles:
         launchMsg = launchFile(package, fileName)
     else:
-        launchMsg = killLaunchFile(package, fileName)
+        launchMsg = killLaunchFile(fileName)
     return launchMsg.message, launchMsg.isLaunched, launchMsg.fileName
 
 def listCallback(data):
-    return launchedFiles.values(), launchedFiles.keys()
+    #Get array of package names
```

Comment:
Humm the function name is `listCallback`, but we are trying to retrieve the package names? Does that package have multiple names, or are there many packages?

Reply:
Maybe something like `GetLaunchFilesPackageName` ?

Reply:
`listCallback` returns a list of every launch file and its package name that are currently launched so the UI can keep track of the status of each one. I could change the name to `GetLaunchedFiles` to make it easier to understand.

Reply:
Yeah, that's more representative of what the function is doing. Because, I don't see any callbacks in the function body ^^.